src/transformState.js

```diff
@@ -5,7 +5,34 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const type = action.type;
+
+    switch (type) {
+      case 'addProperties':
```

Comment:
from my point of view, it is better to write `case something: {do something}` using {}

Reply:
It can depends on code style in the project, so it's ok to write in both styles ✅